docs/Getting-Started.md

```diff
@@ -56,61 +44,167 @@ fastify.listen(3000, function (err) {
 })
 ```
 
-*To learn more about using serialization and validation, see [Validation and Serialization](https://github.com/fastify/fastify/blob/master/docs/Validation-and-Serialization.md)!*
+Awesome, that was easy.  
+Unfortunately, writing a complex application requires significantly more code than this example. A classic problem when you are building a new application is how handle multiple files, asynchronous bootstrapping and the architecture of your code.  
+Fastify offers an easy platform that helps solve all of problems, and more.
```

Comment:
`that helps solve` => `that helps solving`

Reply:
Or add a "these" between "of" and "problems".